From 87c70a9a2828fa9723360ffbcc7b261e9a8d1bb4 Mon Sep 17 00:00:00 2001
From: Florian Dollinger <atar-axis@users.noreply.github.com>
Date: Sun, 14 Apr 2019 14:26:39 +0200
Subject: [PATCH 04/29] Document (final) assignment on the RUN (#12310)

As described in #12291, final assignments and assignments are clearing both command types.

(cherry picked from commit 967de8faceaa83c11a1215515cb135d7a8c0c32c)
---
 man/udev.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/man/udev.xml b/man/udev.xml
index 5a78be3208..98d17bbb54 100644
--- a/man/udev.xml
+++ b/man/udev.xml
@@ -437,6 +437,8 @@
             <para>Note that running programs that access the network or mount/unmount
             filesystems is not allowed inside of udev rules, due to the default sandbox
             that is enforced on <filename>systemd-udevd.service</filename>.</para>
+            <para>Please also note that <literal>:=</literal> and <literal>=</literal> are clearing
+            both, program and builtin commands.</para>
           </listitem>
         </varlistentry>
 
-- 
2.21.0

